content/en/blog/2019/trustworthy-jwt-sds/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Change in Secret Discovery Service in Istio 1.3
+description: Taking advantage of Kubernetes trustworthy JWTs to issue certificates for workload instances more securely.
+publishdate: 2019-09-10
+attribution: Phillip Quy Le (Google)
+keywords: [security, PKI, certificate, nodeagent, sds]
+---
+
+In Istio 1.3, we are taking advantage of improvements in Kubernetes to issue certificates for workload instances more securely.
+
+When a Citadel Agent sends a certificate signing request to Citadel to get a certificate for a workload instance,
+it includes the JWT that the Kubernetes API server issued representing the service account of the workload instance.
+If Citadel can authenticate the JWT, it extracts the service account name needed to issue the certificate for the workload instance.
+
+Before Kubernetes 1.12, the Kubernetes API server issues JWTs with the following issues:
+
+1. The tokens don't have important fields to limit their scope of usage, such as `aud` or `exp`. See [Bound Service Tokens](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/bound-service-account-tokens.md) for more info.
+1. The tokens are mounted onto all the pods without a way to opt-out. See [Service Account Token Volumes](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/svcacct-token-volume-source.md) for motivation.
+
+Kubernetes 1.12 introduces `trustworthy` JWTs to solve these issues.
+However, Kubernetes support for the `aud` field to be different than the API server audience became available until [Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md).
```

Comment:
```suggestion
However, Kubernetes' support for the `aud` field to be different than the API server audience became available until [Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md).
```

Reply:
There's no need to have Kubenetes twice in the sentence also fixing minor verb tense issue. 
```suggestion
However, support for the `aud` field to have a different value than the API server audience didn't become available with [Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md).
```

Reply:
@rcaballeromx The language is messing with me here. So the information that I'm trying to communicate here is k8s didn't support different audience until version 1.3. Is what you're suggesting saying that? I think I missed the `didn't` part in my current PR.

Reply:
Sorry wrong preposition crept in. Is this better:
```suggestion
However, support for the `aud` field to have a different value than the API server audience didn't become available until [Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md).
```